### PR TITLE
Add instructions for installing rustup-toolchain-install-master

### PR DESCRIPTION
**Pull request by chengr4**
When I run `./target/release/collector profile_local XXXXXXXX`, it also asked me to install the tool.

### CODE REVIEW

**Review by Kobzol — COMMENTED** [review]
Hi, thanks for the PR! It depends not so much on the command that you execute, but more on the toolchain type that you use, if you give a SHA, you need this tool.

So claiming that you need the tool for `bench_local`/`profile_local` everytime is a bit misleading. Maybe we could instead specify on a single place (e.g. at the beginning of the section with the available commands) that if you want to specify a toolchain with a commit SHA, you will need this tool.

**Comment by chengr4** [comment]
Thanks for clarifying. In that case, I believe the original one is good enough. I've made the updates and just adjusted the style a bit. What do you think?

File: collector/README.md [hunk 1]
```diff
@@ -91,9 +91,9 @@ The following arguments are mandatory.
   `+f7bb8e3677ba4277914e85a3060e5d69151aed44` in which case `rustup` will be
   used to obtain the toolchain, downloading it if necessary. The commit SHA
   of the toolchain does not need to be in the `rustc` `master` branch, it can be e.g.
-  the result of a `try` CI run. You will need to have
-  [`rustup-toolchain-install-master`](https://github.com/kennytm/rustup-toolchain-install-master)
-  installed in order for this to work.
+  the result of a `try` CI run.
+
+    > If you want to specify a toolchain with a commit SHA, You will need to have [`rustup-toolchain-install-master`](https://github.com/kennytm/rustup-toolchain-install-master) installed.
```
Comment (Kobzol) [inline collector/README.md:96]:
```suggestion
    > If you want to specify a toolchain with a commit SHA, you will need to have [`rustup-toolchain-install-master`](https://github.com/kennytm/rustup-toolchain-install-master) installed.
```

**Review by Kobzol — APPROVED** [review]
Ok, good! :) One small typo and I'll merge it.

**Pull request merged**